lapack/testlapack/general.go

```diff
@@ -1478,9 +1479,8 @@ func dlange(norm lapack.MatrixNorm, m, n int, a []float64, lda int) float64 {
 		}
 	case lapack.Frobenius:
 		for i := 0; i < m; i++ {
-			for j := 0; j < n; j++ {
-				value = math.Hypot(value, a[i*lda+j])
-			}
+			row := f64.L2NormInc(a[i*lda:], uintptr(n), 1)
```

Comment:
If you're just doing a row at a time, I would use `f64.L2NormUnitary`.

Reply:
... or `floats.Norm(a[i*lda:], 2)` which ends up making the same call (if there's not already a dependency on floats, I use the f64 call directly.

Reply:
Of course. I'm not focused.